modern/src/settings/DevicePage.js

```diff
@@ -150,10 +150,15 @@ const DevicePage = () => {
               </AccordionSummary>
               <AccordionDetails className={classes.details}>
                 <DropzoneArea
+                  dropzoneText={t('sharedDropzoneText')}
                   acceptedFiles={['image/*']}
                   filesLimit={1}
                   onChange={handleFiles}
                   alertSnackbarProps={components.MuiSnackbar.defaultProps}
+                  getFileLimitExceedMessage={(filesLimit) => `${t('sharedMaxFileLimit')} ${t('sharedOnly')} ${filesLimit} ${t('sharedAllowed')}`}
+                  getFileAddedMessage={(fileName) => `${t('sharedFile')} ${fileName} ${t('sharedSuccessfullyAdded')}`}
+                  getFileRemovedMessage={(fileName) => `${t('sharedFile')} ${fileName} ${t('sharedRemoved')}`}
+                  getDropRejectMessage={(file) => `${t('sharedFile')} ${file.name} ${t('sharedDropRejectFile')} ${file.size} ${t('sharedMegabytes')}`}
```

Comment:
Is there a way to just disable these toasts?

Also, combining a string from different words like this doesn't really work for translation. Different languages might have different order etc.

Reply:
Yesn Anton i can disable this.